.github/workflows/windows.yaml

```diff
@@ -1,4 +1,4 @@
-name: E2E Podman functionality through PD Podman installation
+name: Podman E2E with Podman installation
```

Comment:
probably rename the file as well ?

Reply:
@benoitf true!, done. Would like to get @adrianriobo opinion as well.